internal/rbd/mirror.go

```diff
@@ -30,13 +30,13 @@ import (
 func (ri *rbdImage) enableImageMirroring(mode librbd.ImageMirrorMode) error {
 	image, err := ri.open()
 	if err != nil {
-		return fmt.Errorf("failed to open image %q with error: %w", ri.String(), err)
```

Comment:
same as in `encryption.go`, `ri` is pointer to `rbdImage`. All of them are inside methods for `(ri *rbdImage)`.